app/redux/sagas/InitializedAnalyticsSaga.js

```diff
@@ -18,9 +20,17 @@ const InitializedAnalyticsSaga = function * InitializedAnalyticsSaga() {
     } else {
         var numWorkoutSets = SetsSelectors.getNumWorkoutSets(state);
     }
+
+    const e1RMLastCalcs = AnalysisSelectors.getE1RMCalcs(state);
+    const e1RMLastCalcCount = AnalysisSelectors.getE1RMCalcsCount(state);
+
     Analytics.logEventWithAppState('initialized', {
         num_workout_sets: numWorkoutSets,
+        one_rm_last_calculate_count: e1RMLastCalcCount,
+        one_rm_last_calculate: e1RMLastCalcs,
```

Comment:
- [ ] it's not supposed to be a parameter on an event, it is ACTUALLY an event by itself that is run after STORE_INITIALIZED is run